.appveyor.yml

```diff
@@ -0,0 +1,27 @@
+# Builds are whole numbered
+version: '{build}'
+
+init:
+  - git config --global core.autocrlf input
```

Comment:
Ok, I think it makes more sense to remove the line. We can add it in the future.